Assets/Code/Scripts/Fight/Grid/FightHexGrid.cs

```diff
@@ -0,0 +1,60 @@
+using FrostfallSaga.Core.GameState;
+using FrostfallSaga.Core.GameState.Fight;
+using FrostfallSaga.Dungeon;
+using FrostfallSaga.Fight.FightCells;
+using FrostfallSaga.Grid;
+using FrostfallSaga.Utils;
+using UnityEngine;
+
+namespace FrostfallSaga.Fight
+{
+    public class FightHexGrid : AHexGrid
+    {
+        [SerializeField] private BiomeTypeSO defaultBiomeType;
+        private DungeonGridGenerator _dungeonGridGenerator;
+        private FightGridGenerator _fightGridGenerator;
+        private FightCell _hexFightPrefab;
+
+        public override void GenerateGrid()
+        {
+            if (Cells.Count > 0)
```

Comment:
```suggestion
            if (Cells.Count > 0)
```
à incorporer dans la méthode clearcells